pkg/storage/cacher_test.go

```diff
@@ -335,7 +338,6 @@ func TestFiltering(t *testing.T) {
 	}
 	defer watcher.Stop()
 
-	verifyWatchEvent(t, watcher, watch.Added, podFoo)
```

Comment:
I actually want to test the normal scenario, passing in a resourceVersion and making sure we don't get that event. We need to exercise the cache skipping the matching resourceVersion event. If we passed resourceVersion-1, we wouldn't know if the cache was skipping or just returning all events it had